src/me/hkgumbs/tictactoe/main/java/simulation/SizeConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package me.hkgumbs.tictactoe.main.java.simulation;
+
+import java.util.List;
+
+public class SizeConfiguration implements Configuration {
+
+    private static final String KEY = "--size";
+    private static final int DEFAULT_SIZE = 3;
+
+    private int extractSize(List<String> args) {
```

Comment:
This method looks pretty much identical to `extractPadding` in `FormatterConfiguration`. Surely there's a way to DRY that up - maybe a small helper class with (at least) one method that takes the argument list, the argument name, and the default value?